com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/pywrapfunction.cgt

```diff
@@ -14,24 +14,30 @@
     PyObject * appCallable = 
       streamsx::topology::Splpy::loadFunction(<%=$pyModule%>, <%=$pyCallableName%>);
 
+	<% if ($pystyle eq 'string' or $pystyle eq 'message') { %>
+		function_ = appCallable;
+	<%} else {%>	  
     // The object to be called is either appCallable for
     // a function passed into the operator
     // or a pickled encoded class instance
     // represented as a string in parameter pyCallable
-    
-    <% if ($pyCallable) { %>
-      // argument is the serialized callable instance
-      Py_DECREF(appCallable);
-      appCallable = Py_BuildValue("s", <%=$pyCallable%>);
-    <%}%>
+    	<% if ($pyCallable) { %>
```

Comment:
Try to avoid whitepace changes, it makes it harder to review a change. 

Doesn't seem like there are changes here?????